src/main/java/formflow/library/ValidationService.java

```diff
@@ -158,4 +163,25 @@ private Map<String, List<String>> performFieldLevelValidation(String flowName, F
 
     return validationMessages;
   }
+
+  public static Map<String, Boolean> getRequiredInputs(String flowName) {
+    if (requiredInputs.isEmpty()) {
+      Class<?> flowClass;
+
+      try {
+        flowClass = Class.forName(inputConfigPath + StringUtils.capitalize(flowName));
+      } catch (ReflectiveOperationException e) {
+        throw new RuntimeException(e);
+      }
```

Comment:
I think we may want more logging here. This implies a major configuration issue if we fail in this section - is that right?   (Or a bad flow name, which seems unlikely)

Reply:
Added